webcompat/webhooks/helpers.py

```diff
@@ -140,11 +157,11 @@ def new_opened_issue(payload):
     issue_body = payload.get('issue')['body']
     issue_number = payload.get('issue')['number']
     browser_label = extract_browser_label(issue_body)
+    extra_label = extract_extra_label(issue_body)
     priority_label = extract_priority_label(issue_body)
-    if browser_label:
-        labels.append(browser_label)
-    if priority_label:
-        labels.append(priority_label)
+    for label in (browser_label, extra_label, priority_label):
+        if label:
+            labels.append(label)
```

Comment:
so here it smells list comprehension :)
Let's remove the top `labels = []`

```python
>>> # Let's define the labels.
>>> browser_label = 'browser-firefox'
>>> extra_label = None
>>> priority_label = 'priority-important'
>>> # Without an if, we would get None in the list.
>>> labels = [label for label in (browser_label, extra_label, priority_label)]
>>> labels
['browser-firefox', None, 'priority-important']
>>> # Not what we want, but we can test for this with a conditional list comprehension
>>> labels = [label for label in (browser_label, extra_label, priority_label) if label is not None]
>>> labels
['browser-firefox', 'priority-important']
>>> # Yeah. Oh and if you are worried about the case where everything is None. We got you covered here.
>>> [label for label in (None, None, None) if label is not None]
[]
```

Reply:
Heh, I think nearly every Python review has some instance of me not remembering to use a list comprehension. :)